src/steps/05-PerformanceRequirements/DOW/ComputeForm.spec.ts

```diff
@@ -0,0 +1,700 @@
+        })
+      });
+    });
+  
+    describe("Classification Label", () => {
+      it("tests if expected verbiage mentions expected single classification level", async () => {
+        await wrapper.setData({
+          avlClassificationLevelObjects: [avlClassificationLevelObjects[0]]
+        })
+        const spy = await jest.spyOn(wrapper.vm, "checkSingleClassification");
+        await wrapper.vm.checkSingleClassification();
+        expect(spy).toHaveBeenCalled();
+        const classLevelName = wrapper.vm.$data.singleClassificationLevelName;
+        expect(classLevelName).toContain(
+          wrapper.vm.$data.avlClassificationLevelObjects[0].impact_level
+        )
+      })
+    });
+
+
+    describe("Classification Modal Interactions", () => {
+      it("tests if modal classification levels are changed `modalSelectedOptions` then "+
+         "available Classification Levels `avlClassificationLevelObjects' are same " +
+         "avlClassificationLevelObjects[] === modalSelectedOptions ", async () => {
+
+        await wrapper.setData({
+          allClassificationLevels: allClassificationLevels,
+          modalSelectedOptions: ["class3", "class4"]
+        })
+
+        //closes modal and sets avlClassificationLevelObjects with 
+        //class levels selected in modal
+        const spy = await jest.spyOn(wrapper.vm, "classificationLevelsChanged");
+        await wrapper.vm.classificationLevelsChanged();
+        expect(spy).toHaveBeenCalled();
+        
+        //expects modal to be closed and data props to be set accordingly
+        expect(wrapper.vm.$data.showDialog).toBe(false);
+        expect(wrapper.vm.$data.avlClassificationLevelObjects.length).toEqual(2);
+
+        // available class levels should be same as 
+        // modal selected options
+        Vue.nextTick(()=>{
+          expect(wrapper.vm.$data.avlClassificationLevelObjects).toEqual(
+            wrapper.vm.$data.modalSelectedOptions
+          )
+        })
+      })
+
+      it("tests if only one option was selected in classification levels modal then" +
+         "checkSingleClassification is to be called", async () => {
+        
+        await wrapper.setData({
+          allClassificationLevels: allClassificationLevels,
+          modalSelectedOptions: ["class3"]
+        })
+
+        //closes modal and sets avlClassificationLevelObjects with 
+        //class levels selected in modal
+        const classLevelsChangedSpy = await jest.spyOn(wrapper.vm, "classificationLevelsChanged");
+        await wrapper.vm.classificationLevelsChanged();
+        expect(classLevelsChangedSpy).toHaveBeenCalled();
+        
+        //expects modal to be closed and data props to be set accordingly
+        expect(wrapper.vm.$data.showDialog).toBe(false);
+        expect(wrapper.vm.$data.avlClassificationLevelObjects.length).toEqual(1);
+
+        const checkSingleClassSpy = await jest.spyOn(wrapper.vm, "checkSingleClassification");
```

Comment:
These last 3 lines aren't really doing anything (see my other comments on testing a test)

Reply:
Removed all references to spy as they were not needed as we are calling the functions directly